### PR TITLE
JBPM-6110 - Adding support to export a canvas context to SVG

### PR DESCRIPTION
**Pull request by tiagodolphine**
_Stunner_ needs to export diagrams presented on canvas to SVG files, the idea is to add this feature and an abstraction of canvas and the library implementation used for that on _Uberfire,_ in this way any other components that need to export a canvas to SVG may use this feature.

* Added a `FileExport` implementation to support SVG on `SvgFileExport`
* Abstraction interface to the Context2D that is used to generate the SVG
* _JsInterop_ to the library **[canvas2svg](https://github.com/gliffy/canvas2svg)** with an overlay to distinguish save/restore from groups, that is the containers nodes, the idea is to open a PR to add this feature on the native canvas2svg, then we could remove the overlay
* _Webjar_ configuration to **canvas2svg**

Related JIRA https://issues.jboss.org/browse/JBPM-6110

@romartin 
@mdproctor

### CODE REVIEW

**Comment by romartin** [comment]
@tiagodolphine missing the ticket id in the commit message, plz :+1:  Or at lest once merging it, please @ederign remember to add it... thx guys!

File: uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/exports/jso/FileExportScriptInjector.java [hunk 1]
```diff
@@ -37,6 +39,7 @@
     public static final String NS = "org.uberfire.ext.editor.commons.client.file.exports.jso.";
     public static final String NS_SEPARATOR = ".";
     public static final String JS_OBJ_SUFFIX = " || {};";
+    private static Logger LOGGER = Logger.getLogger(FileExportScriptInjector.class.getName());
```
Comment (romartin) [inline uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/exports/jso/FileExportScriptInjector.java]: added but unused...

**Comment by tiagodolphine** [comment]
@ederign @romartin I will update the commit message and the webjar dependency as well. I'll let you know when the changes have been pushed.

**Comment by tiagodolphine** [comment]
Jenkins retest this

**Pull request merged**